third_party/move/move-compiler-v2/tests/acquires-checker/copy_ability_tuple.exp

```diff
@@ -0,0 +1,7 @@
+
+Diagnostics:
+error: unnecessary acquires annotation
+  ┌─ tests/acquires-checker/copy_ability_tuple.move:8:39
+  │
+8 │     public fun g(s: &signer) acquires R {
```

Comment:
Can we dig into the parser and see why the `^^` extends beyond the specifier?

Reply:
filed issue https://github.com/aptos-labs/aptos-core/issues/12761